migrations/20221101_insert_into_accounts_ryan_accounts.sql

```diff
@@ -1 +1 @@
-INSERT INTO accounts (login, password) VALUES ('ryan', 'abc123');
\ No newline at end of file
+INSERT INTO accounts (login, password) VALUES ('ryan', 'c70b5dd9ebfb6f51d09d4132b7170c9d20750a7852f00680f65658f0310e810056e6763c34c9a00b0e940076f54495c169fc2302cceb312039271c43469507dc');
```

Comment:
Which clean password ryan has? That kind of database operations are not schema change migrations, but fake data to test, so must be removed from migrations folder and put on test_data or something like that. After that, must be refactored to use temporary table (i.e. WITH statement) to become clean the password.

Reply:
I didn't understand what is a clean password. Could you please explain it better or give me some examples or documentation?

Reply:
clean password = abc123. We will lost them when make that change. 

the use of WITH is postgresql-specific, so add a comment to indicate that c70b5dd9ebfb6f51d09d4132b7170c9d20750a7852f00680f65658f0310e810056e6763c34c9a00b0e940076f54495c169fc2302cceb312039271c43469507dc is the hash of abc123.

Then make yoyo down to remove your migrations from database. Then move that migration to a folder "fake_data". Now reapply migrations. Now you can apply the fake data outside migrations, so when we put on production, the fake data will not be there.